Resources/fsh-generated/resources/Appointment-ISiKTerminExampleExtendedICU.json

```diff
@@ -0,0 +1,75 @@
+{
+  "resourceType": "Appointment",
+  "id": "ISiKTerminExampleExtendedICU",
+  "meta": {
+    "tag": [
+      {
+        "code": "external",
+        "system": "http://fhir.de/CodeSystem/common-meta-tag-de"
+      }
+    ],
+    "profile": [
+      "https://gematik.de/fhir/isik/v2/Terminplanung/StructureDefinition/ISiKTermin"
+    ]
+  },
+  "extension": [
+    {
+      "url": "https://gematik.de/fhir/isik/v2/Terminplanung/StructureDefinition/ISiKNachrichtExtension",
+      "valueReference": {
+        "reference": "Communication/ISiKNachrichtExample"
+      }
+    }
+  ],
+  "status": "proposed",
+  "start": "2022-12-10T09:00:00Z",
+  "end": "2022-12-10T09:30:00Z",
+  "slot": [
+    {
+      "reference": "ISiKSlotExample"
+    }
+  ],
+  "serviceType": [
+    {
+      "coding": [
+        {
+          "code": "174",
+          "system": "http://terminology.hl7.org/CodeSystem/service-type"
+        }
+      ]
+    }
+  ],
+  "specialty": [
+    {
+      "coding": [
```

Comment:
issue: @f-peverali habe zu spät gesehen, aber hier wurde nach dem Kompilieren ein coding weggelassen (AerztlicheFachrichtung)! ggf. ist die Syntax in der fsh-Datei falsch!